client/hosting/sites/components/sites-dataviews/dataviews-fields/site-field.tsx

```diff
@@ -80,8 +80,8 @@ const SiteField = ( { site, openSitePreviewPane }: Props ) => {
 		event.preventDefault();
 	};
 
-	const isMigrating = isMigrationInProgress( site );
-	const siteTitle = isMigrating ? translate( 'Incoming Migration' ) : site.title;
+	const isMigrationPending = getMigrationStatus( site ) === 'pending';
+	const siteTitle = isMigrationPending ? translate( 'Incoming Migration' ) : site.title;
```

Comment:
This fixes the site title being updated to "Incoming Migration" when the migration status is "started". In the design, it's updated only when the status is "pending"